src/SFA.DAS.RoatpGateway.Web/Infrastructure/ApiClients/RoatpExperienceAndAccreditationApiClient.cs

```diff
@@ -33,6 +33,16 @@ public async Task<FileStreamResult> GetSubcontractorDeclarationContractFile(Guid
             return result;
         }
 
+        public async Task<FileStreamResult> GetSubcontractorDeclarationContractFileClarification(Guid applicationId, string fileName)
+        {
+            var response = await GetResponse($"/Accreditation/{applicationId}/SubcontractDeclaration/ContractFileClarification/{fileName}");
```

Comment:
That's quite a nasty URL for an end point  but I'm guessing our options to make it more human readable are very limited